$OpenBSD$

warning: passing argument 3 of 'eina_xattr_get' from incompatible pointer type

--- src/lib/eet_data.c.orig	Wed Oct 26 01:47:58 2011
+++ src/lib/eet_data.c	Wed Oct 26 01:49:22 2011
@@ -4892,7 +4892,7 @@ eet_data_xattr_cipher_get(const char *filename,
 {
    void *blob;
    void *ret;
-   int size;
+   ssize_t size;
 
    blob = eina_xattr_get(filename, attribute, &size);
    if (!blob) return NULL;
